src/lib/dialog/dialog.md

```diff
@@ -29,6 +29,37 @@ Components created via `MdDialog` can _inject_ `MdDialogRef` and use it to close
 in which they are contained. When closing, an optional result value can be provided. This result
 value is forwarded as the result of the `afterClosed` promise. 
 
+### Sharing Data with the Dialog component.
+Depending on what you are doing you might want to share data with your dialog component. The dialog component has a data option that you can include in order to pass data from the open to the dialog. 
+
+Passing outside data to your component is as simple as.
+```ts
+import {MdDialog} from '@angular/material';
+
+private dialog: MdDialog;
+
+let dialogRef = dialog.open(DialogName, {
+  data:'your data',
+});
+```
+
+Here is an example component you can pass data to.
+```ts
+import {Component, Inject} from '@angular/core';
+import {MD_DIALOG_DATA} from '@angular/material';
+
+@Component({
+  selector: 'dialog-selector',
```

Comment:
Try renaming this to something like `your-dialog` and the class to `YourDialog`.